src/main/resources/META-INF/plugin.xml

```diff
@@ -19,17 +19,26 @@
     Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
 
 -->
-<idea-plugin url="http://intellij.sonarlint.org/">
+<idea-plugin url="http://www.sonarlint.org">
     <id>org.sonarlint.idea</id>
     <name>SonarLint</name>
     <version>${project.version}</version>
     <vendor email="sonarlint@googlegroups.com" url="http://www.sonarsource.com">SonarSource</vendor>
```

Comment:
@dbmeneses : do you know if I can just dump the _email_ attribute here, or is it mandatory to have one ?

Reply:
You probably can, I see other plugins without it.